File: python/p010.py
def isPrime(p):
    for j in range(2, int(1+p**0.5)):
        if p%j == 0:
            return False
    return True

s = 0
for n in range(2, 2 * 10**6):
    if isPrime(n):
        s += n

print(s)

File: cpp/p010.cpp
#include <iostream>
#include <math.h>
using namespace std;

bool isPrime(int n);

int main() {
    long int s = 0;
    for(int p = 2; p < 2000000; p++) {
        if(isPrime(p))
            s += p;
    }
    cout << "Total: " << s << '\n';
    return 0;
}

bool isPrime(int n) {
    for(int d = 2; d <= sqrt(n); d++) {
        if(n%d == 0)
            return false;
    }
    return true;
}
